src/dune_engine/package.ml

```diff
@@ -227,34 +231,35 @@ module Dependency = struct
     <|> let+ name = Name.decode in
         { name; constraint_ = None }
 
-  let rec opam_constraint : Constraint.t -> OpamParserTypes.value =
-    let nopos = Opam_file.nopos in
+  let nopos pelem = OpamParserTypes.FullPos.{ pelem; pos = Opam_file.nopos }
```

Comment:
`{ OpamParserTypes.FullPos.pelem; pos = Opam_file.nopos }`

Reply:
Fixed and moved the whole function one up into the module, since I had it defined multiple times.